lesson1/serhii_khrychov_hometask.rb

```diff
@@ -0,0 +1,33 @@
+class Foo
+
+  def initialize(uniq = :not_uniq)
+    @@counter ||= 0
+    @@counter += 1
+    @instance_num = @@counter
+    @uniq = uniq
+  end
+
+  def my_instance
+    return :uniq if @uniq == :uniq
+    @instance_num.even? ? :even : :odd
+  end
+end
+
+instances = []
+
+uniq_instance = Foo.new(:uniq)
+
+for i in 0..5
```

Comment:
Do not use `for` in ruby at all. I forgot to write this first time, but it's a bad practice. Your case could be resolved by [#times](https://ruby-doc.org/core-2.2.0/Integer.html#method-i-times)